### DISCUSSION TITLE
Facing error in opening any execrcise

### DISCUSSION THREAD

**Discussion by lavish123409** [JdeRobot/RoboticsAcademy#2436] [General]
Hello everyone,

I have recently set up the repo. Now, when I am trying to run any exercise, I am receiving this error `Invalid roslaunch XML syntax: [Errno 2] No such file or directory: '/opt/jderobot/Launchers/{exercide_name}.launch.py'`.

For follow line exercise, I manually copied the file from `/RoboticsAcademy/exercises/static/exercises/python_web_ide/launch/ros2_humble/simple_circuit_followingcam.launch.py` to `/opt/jderobot/Launchers/`. But, then I am receiving this error `Invalid roslaunch XML syntax: syntax error: line 1, column 0`.

I have just changed the version of node to 20x while building RADI and rest of the procedure is exactly same. Did I missed something in setting up the developer environment or something else?

Thanks

**Comment by jmplaza** [comment]
Uhm... you mean with your own built miniRADI, right? The developer workflow seems to be broken...

Does it work RoboticsAcademy on your machine when using RADI 3.4.30 (=ROS1-Noetic based) or RADI 4.4.32 (=ROS2-Humble based)? They are both downloadable from [DockerHub](https://hub.docker.com/r/jderobot/robotics-academy/tags).

We are now pretty busy on the upcoming RoboticsAcademy 5.2 release, which has significant changes. Once we publish it we will refine the developer workflow and its documentation. Just wait a little bit. That 5.2 release will be the baseline for the RoboticsAcademy projects in GSoC-2024.

**Reply by lavish123409** [reply]
Yes, I tried with RADI 3.4.30 image, and it is working fine.

> We are now pretty busy on the upcoming RoboticsAcademy 5.2 release, which has significant changes. Once we publish it we will refine the developer workflow and its documentation. Just wait a little bit. That 5.2 release will be the baseline for the RoboticsAcademy projects in GSoC-2024.

Ok, not an issue, but till then Is there something that I can work on?
